### PR TITLE
Added Open Graph meta tags

### LINKED ISSUE #3675: Add OpenGraph meta tags
**Is your feature request related to a problem? Please describe.**

Not a problem. Just add Open Graph meta tags for title, URL, image, decs, and type to the website.

*The Goals*
- Make the content more eye-catching in social media feeds.
- Tell people what the content is about at a glance.
- Sharing via social media helps this project grow bigger

### PR DESCRIPTION
**Pull request by danangtomo**
## :bookmark_tabs: Summary

Add Open Graph meta tags to the website.

Resolves #3675 

### :clipboard: Tasks

Make sure you

- [x] :book: have read the [contribution guidelines](https://github.com/mermaid-js/mermaid/blob/develop/CONTRIBUTING.md)
- [ ] :computer: have added unit/e2e tests (if appropriate)
- [x] :bookmark: targeted `develop` branch

### CODE REVIEW

File: packages/mermaid/src/docs/index.html [hunk 1]
```diff
@@ -15,6 +15,11 @@
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     />
+    <meta property="og:title" content="Mermaid" />
+    <meta property="og:url" content="https://mermaid-js.github.io/mermaid/" />
+    <meta property="og:image" content="https://i.ibb.co/883k0pH/mermaid-logo.jpg" />
```
Comment (aloisklink) [inline packages/mermaid/src/docs/index.html:20]:
```suggestion
    <meta property="og:image" content="https://mermaid-js.github.io/mermaid/img/header.png" />
```

My gut feeling is that we should instead be using https://mermaid-js.github.io/mermaid/img/header.png as an image. What do you think?

**Review by aloisklink — CHANGES_REQUESTED** [review]
It looks mostly good to me, but I think we can use a better image.

Also, you'll probably need to run the following to fix the Lint CI issue:

```bash
pnpm install
pnpm run lint:fix
pnpm run docs:build
```

File: packages/mermaid/src/docs/index.html (hunk 1 above)
Comment (danangtomo) [reply packages/mermaid/src/docs/index.html:20]: looks good, the picture better explains the use of the mermaid. great

Comment (danangtomo) [reply packages/mermaid/src/docs/index.html]: > My gut feeling is that we should instead be using https://mermaid-js.github.io/mermaid/img/header.png as an image. What do you think?

done

**Review by aloisklink — APPROVED** [review]
It looks good to me!

Since it's just a documentation change, it should be okay if it's only me that reviews this before merging, but I'll leave this open for a few days in case anybody else wants to comment.

**Comment by danangtomo** [comment]
> It looks good to me!
> 
> Since it's just a documentation change, it should be okay if it's only me that reviews this before merging, but I'll leave this open for a few days in case anybody else wants to comment.

may i ask you to add the label 'hacktoberfest-accepted'  ?

**Comment by danangtomo** [comment]
okay, thanks 🙌 @aloisklink

**Comment by danangtomo** [comment]
sorry I accidentally deleted my repo but now it's open again✌

**Review by sidharthv96 — APPROVED** [review]

**Pull request merged**